File: src/main/java/pers/lyks/spring/example/ServletApplication.java
package pers.lyks.spring.example;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

/**
 * Spring-boot web application main method.
 *
 * @author lawyerance
 * @version 1.0 2018-08-03
 */
@SpringBootApplication
public class ServletApplication {
    public static void main(String[] args) {
        SpringApplication application = new SpringApplication(ServletApplication.class);
        application.run(args);
    }
}

File: src/main/kotlin/pers/lyks/spring/example/ServletApplication.kt
package pers.lyks.spring.example

import org.mybatis.spring.boot.autoconfigure.MybatisAutoConfiguration
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication

/**
 *
 * @author lawyerance
 * @version 1.0 2019-09-22
 */
@SpringBootApplication(exclude = [MybatisAutoConfiguration::class])
open class ServletApplication

fun main(args: Array<String>) {
    runApplication<ServletApplication>(*args)
}


